linux_sql/scripts/host_usage.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
+
```

Comment:
same comments from host_info.sh apply here as well: return required params to user, and fix vmstate name